pkg/visor/visor.go

```diff
@@ -57,11 +58,13 @@ type Visor struct {
 	ebc   *appevent.Broadcaster // event broadcaster
 	dmsgC *dmsg.Client
 
-	tpM      *transport.Manager
-	arClient addrresolver.APIClient
-	router   router.Router
-	rfClient rfclient.Client
+	stunClient *network.StunDetails
+	tpM        *transport.Manager
+	arClient   addrresolver.APIClient
+	router     router.Router
+	rfClient   rfclient.Client
 
+	isNetConf   chan bool             // net config check
```

Comment:
Naming here is not clear to me. What are we doing with the value?

Reply:
That was leftover code. Will take it out.